bootstrap/release/linux.py

```
@@ -0,0 +1,245 @@
+    """Fixpoint: bundle every resolvable lib not yet in the AppDir.
+
+    Skips the dlopen'd graphics-driver family, which must come from the host.
+    """
+    usr_lib = appdir / "usr" / "lib"
+    targets = [appdir / "usr" / "bin" / BINARY_NAME]
+    targets.extend(sorted((appdir / "usr" / "bin" / "lib").glob("*.so*")))
+    lib_path = f"{appdir}/usr/bin/lib:{usr_lib}:{search_path}"
+    while True:
+        missing: dict[str, Path] = {}
+        for f in targets + sorted(usr_lib.glob("*.so*")):
+            out = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)